library/src/main/java/com/novoda/downloadmanager/MigrationJob.java

```diff
@@ -218,4 +219,38 @@ private void deleteVersionOneDatabase(InternalMigrationStatus migrationStatus, S
         database.deleteDatabase();
     }
 
+    private void migrateV1FilesToV2Location(FilePersistence filePersistence, Migration migration) {
```

Comment:
Turns out we still need to move the files 😂 we need them to appear per batch rather than globally, which is what v1 was doing.